lib/live_debugger/api/traces_storage.ex

```diff
@@ -449,31 +461,62 @@ defmodule LiveDebugger.API.TracesStorage do
 
       execution_times = Keyword.get(opts, :execution_times, %{})
       node_id = Keyword.get(opts, :node_id)
+      trace_diffs = Keyword.get(opts, :trace_diffs, false)
 
-      match_spec(node_id, functions, execution_times)
+      match_spec(node_id, functions, execution_times, trace_diffs)
```

Comment:
Why not just do this


```suggestion
      match_spec(node_id, functions, execution_times)
      |> maybe_attach_diff_spec(trace_diffs)
```